app/controllers/events_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
   def show
     @event = Event.find(params[:id])
+    redirect_to action: :index if @event.in_the_past?
```

Comment:
We'll need to fix this at some point, but it's not strictly necessary for this PR.

Reply:
OK. We'll fix it eventually.